stiles.css

```diff
@@ -1,176 +1,181 @@
-html, body {
-    height: 100%;
-    margin: 0px;
-  }
-  
-  body{
-    margin: 0 auto;
-  }
-  
-  *{
-    box-sizing: border-box;
-  }
-  /* projects  */
-  
-  .header{
-    background-color: red;
-    height: 40px;
-    position: sticky;
-    top: 0;
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    padding-right: 20px;
-  }
-  
-  /*header  */
-  .header-content{
-    display: flex;
-    flex-direction: row;
-    gap: 20px;  
-  }
```

Comment:
Great job doing the project 👍. But, I think it will be better if you use "rem" units instead of using "px" so that the page will look responsive at any screen size. 😊 🛠️